### PR TITLE
Loosen tqdm dependency

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Having too strict of a requirement on `tqdm` itself can cause incompatibilities in downstream package environments that might want to use it; always best for a generic package like this to keep things as loose as possible (perhaps even remove version pinning entirely? Unless this specific injection structure didn't work this way until a certain version of `tqdm`, in which case that is the lower bound)

### CODE REVIEW

**Comment by garrettmflynn** [comment]
[v4.49.0](https://tqdm.github.io/releases/#v4490-2020-09-12) looks like that lower bound. Does that make sense to you?

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-tqdm==4.66.1
+tqdm>=4.66.1
```
Comment (CodyCBakerPhD) [inline requirements.txt]:
```suggestion
tqdm>=4.49.0
```

**Comment by CodyCBakerPhD** [comment]
> [v4.49.0](https://tqdm.github.io/releases/#v4490-2020-09-12) looks like that lower bound. Does that make sense to you?

Oh wow, that's later than I would have expected. Sounds good then

**Review by garrettmflynn — APPROVED** [review]
Perfect. Thanks!

**Pull request merged**